src/NServiceBus.AcceptanceTesting/Customization/BusConfigExtensions.cs

```diff
@@ -0,0 +1,16 @@
+namespace NServiceBus.AcceptanceTesting.Customization
+{
+    using System;
+    using System.Collections.Generic;
+
+    public static class BusConfigExtensions
+    {
+        /// <summary>
+        /// Backdoor into the core types to scan. This allows people to test a subset of functionality when running Acceptance tests
```

Comment:
How does the statement in the comment hold true when we do not release the ATT library? what can people do who are not using it?